test/controllers/scripts/imports_controller_test.rb

```diff
@@ -41,7 +57,18 @@ class Scripts::ImportsControllerTest < ActionController::TestCase
       file: fixture_file_upload('files/test.sh', 'text/plain', false)
     }
 
-    assert_redirected_to scripts_url
-    assert_equal I18n.t('scripts.imports.create.fail'), flash.alert
+    assert_redirected_to scripts_imports_new_url
+    assert_equal I18n.t('scripts.imports.create.file_invalid_extension'),
+                 flash.alert
+  end
+
+  test 'should not import if the zip has invalid json' do
+    post :create, params: {
+      file: fixture_file_upload('files/invalidJson.zip', 'zip', false)
```

Comment:
```suggestion
      file: fixture_file_upload('files/invalid_json.zip', 'zip', false)
```
Preferible el _snake case_ en los nombres de archivo, a no ser que por alguna otra convención lo tengamos que modificar.

Reply:
perdon franco. Me olvido y sigo con la cabeza en c# jajajajaja. Mala mia